fedimint-core/src/encoding/secp256k1.rs

```diff
@@ -36,6 +40,21 @@ impl Decodable for secp256k1::PublicKey {
     }
 }
 
+impl Encodable for secp256k1_29::PublicKey {
+    fn consensus_encode<W: std::io::Write>(&self, writer: &mut W) -> Result<usize, std::io::Error> {
+        self.serialize().consensus_encode(writer)
+    }
+}
+
+impl Decodable for secp256k1_29::PublicKey {
+    fn consensus_decode<D: std::io::Read>(
+        d: &mut D,
+        modules: &ModuleDecoderRegistry,
+    ) -> Result<Self, DecodeError> {
+        Self::from_slice(&<[u8; 33]>::consensus_decode(d, modules)?).map_err(DecodeError::from_err)
+    }
+}
```

Comment:
Note for reviewer: I prefer to migrate implementations of `Encodable` and `Decodable` all at once, but `PublicKey` is used in a lot of crates and I don't want to migrate them all in one giant PR, so we're duplicating the implementation for now